app/scripts/modules/kubernetes/src/v2/serverGroup/details/details.controller.ts

```diff
@@ -35,17 +35,22 @@ class KubernetesServerGroupDetailsController implements IController {
       .catch(() => this.autoClose());
   }
 
-  public canResizeServerGroup(): boolean {
+  public canScaleServerGroup(): boolean {
     return this.serverGroup.kind !== 'DaemonSet';
```

Comment:
It might be nice to have an enum for Kubernetes kinds. Or not.

Reply:
I was thinking of actually having these be discoverable somehow... e.g. a clouddriver endpoint that tells you what is supported.